testsuite/deployments/wildfly-shared/pom.xml

```diff
@@ -22,10 +22,14 @@
         <wildfly-maven-plugin.groupId>org.wildfly.plugins</wildfly-maven-plugin.groupId>
         <wildfly-maven-plugin.artifactId>wildfly-maven-plugin</wildfly-maven-plugin.artifactId>
         <wildfly-maven-plugin.version>4.2.2.Final</wildfly-maven-plugin.version>
-        <!-- Default WildFly `ee` BOMs version is set here and can be overridden for pulling the right BOM -->
+        <!-- Default WildFly `ee` BOM version is set here and can be overridden for pulling the right BOM -->
         <bom.wildfly-ee.groupId>org.wildfly.bom</bom.wildfly-ee.groupId>
         <bom.wildfly-ee.artifactId>wildfly-ee</bom.wildfly-ee.artifactId>
         <bom.wildfly-ee.version>${version.wildfly-server}</bom.wildfly-ee.version>
+        <!-- Default WildFly `microprofile` BOM version is set here and can be overridden for pulling the right BOM -->
+        <bom.wildfly-microprofile.groupId>org.wildfly.bom</bom.wildfly-microprofile.groupId>
+        <bom.wildfly-microprofile.artifactId>wildfly-microprofile</bom.wildfly-microprofile.artifactId>
+        <bom.wildfly-microprofile.version>${version.wildfly-server}</bom.wildfly-microprofile.version>
```

Comment:
@fabiobrz I checked and this would be, in XP5 builds:
```
META_BUILD_BOMS_XP_MICROPROFILE_GAV=org.jboss.bom:jboss-eap-xp-microprofile:5.0.0.GA-redhat-20240304
``` 
and would be propagated to Intersmash as:
```
bom.wildfly-microprofile.groupId=org.jboss.bom
bom.wildfly-microprofile.artifactId=jboss-eap-xp-microprofile
bom.wildfly-microprofile.version=5.0.0.GA-redhat-20240304
```
I suppose it's OK